tests/unit/test_add_constant_to_spline.py

```diff
@@ -0,0 +1,129 @@
+"""
+Tests of `add_constant_to_spline`
+"""
+
+from __future__ import annotations
+
+import numpy as np
+import pytest
+
+from gradient_aware_harmonisation.timeseries import Timeseries
+from gradient_aware_harmonisation.utils import add_constant_to_spline
+
+scipy = pytest.importorskip("scipy")
+
+
+@pytest.mark.parametrize("harmonisation_time", [3.0, 3.5, 4.0])
+def test_constant_to_spline(harmonisation_time):
```

Comment:
```suggestion
def test_add_constant_to_spline(harmonisation_time):
```

There's quite a lot going on in this test. Which bit exactly do you want to test?

Reply:
Yes, it was a poor attempt to find out where the problem lies with the output of the harmonise() function. I adjusted the test now.